### ISSUE TITLE
how could I use  this parameter " linear-lr"?Where could I see it in the origin code?

### ISSUE THREAD

**Issue by yangrisheng** [ultralytics/yolov5#6552]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

how could I use  this parameter " linear-lr"?Where could I see it in the origin code?
Thank you!

**Comment by mpdroid** [comment]
The code is in `train.py`

```
  if opt.linear_lr:                                                                                                                                                                                      
        lf = lambda x: (1 - x / (epochs - 1)) * (1.0 - hyp['lrf']) + hyp['lrf']  # linear
    else:
        lf = one_cycle(1, hyp['lrf'], epochs)  # cosine 1->hyp['lrf']
```

It is the alternative learning rate scheduler in yolo5 similar to [pytorch LinearLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.LinearLR.html).

With this option, I think the learning rate increases  in warmup phase and then decays linearly.

The default I think behaves like a one cycle scheduler - warmup to from `lr0` to `lrf` for three epochs and then reduces gradually to the end.

**Comment by yangrisheng** [comment]
> The code is in `train.py`
> 
> ```
>   if opt.linear_lr:                                                                                                                                                                                      
>         lf = lambda x: (1 - x / (epochs - 1)) * (1.0 - hyp['lrf']) + hyp['lrf']  # linear
>     else:
>         lf = one_cycle(1, hyp['lrf'], epochs)  # cosine 1->hyp['lrf']
> ```
> 
> It is the alternative learning rate scheduler in yolo5 similar to [pytorch LinearLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.LinearLR.html).
> 
> With this option, I think the learning rate increases in warmup phase and then decays linearly.
> 
> The default I think behaves like a one cycle scheduler - warmup to from `lr0` to `lrf` for three epochs and then reduces gradually to the end.

Sry,I’m not reading train.py closely.Thank you!

**Comment by glenn-jocher** [comment]
<img width="598" alt="Screenshot 2022-02-08 at 12 26 11" src="https://user-images.githubusercontent.com/26833433/152978014-7b093865-2c85-478d-b97c-10df99aacba2.png">
